flake8_html/plugin.py

```diff
@@ -179,6 +198,8 @@ def finished(self, filename):
         rendered = self.source_template.render(**params)
         with codecs.open(source_filename, 'w', encoding='utf8') as f:
             f.write(rendered)
+        # stop() isn't called when, f.ex. called via some CI runners, so call it
+        self.stop()
```

Comment:
Looking into the code, I realise that this will rewrite that this change will cause styles, images and index.html to be rewritten for every single file. That's very inefficient and will slow down runs of flake8-html on big codebases significantly.

In what circumstances is stop() not called? Can address that problem instead?